File: src/main/java/com/example/MonaServer/MonaServerApplication.java
package com.example.MonaServer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;

@SpringBootApplication
public class MonaServerApplication {

	public static void main(String[] args) {
		SpringApplication.run(MonaServerApplication.class, args);
	}

}

File: src/main/kotlin/de/lrprojects/monaserver/MonaserverApplication.kt
package de.lrprojects.monaserver

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling


@SpringBootApplication(scanBasePackages = ["de.lrprojects"])
@ConfigurationPropertiesScan
@EnableCaching
@EnableScheduling
class MonaserverApplication
fun main(args: Array<String>) {
    runApplication<MonaserverApplication>(args = args)
}




